### ISSUE TITLE
FAIL on Android 7

### ISSUE THREAD

**Issue by juanpc2018** [slowscript/warpinator-android#114]
Gui works, detects the other machine, but crashes "explodes" when sending a .pdf file.

Kernel 3.10.84
Sep, 2017

**Comment by slowscript** [comment]
I know about this issue and have a general idea where in the code it happens. Unfortunately I'm not sure what is exactly the cause (and the fix) because I cannot replicate it on my devices. It seems to be something vendor specific. I've taken a blind shot and made this update [app-release.zip](https://github.com/slowscript/warpinator-android/files/9551061/app-release.zip). Could you please test it out? It is signed with regular release keys that I use for Google Play and the other APKs here so unless you used F-Droid it should install cleanly. If you can't test it now, could you at least tell me what phone model and Android version you are using? Thank you.

**Comment by slowscript** [comment]
I've been able to reproduce the issue now. Could you please confirm if version 1.5.2 fixes it?

**Comment by juanpc2018** [comment]
Yes & No..,.

Found something interesting...

when i try to send a File, .jpg
Warpinator asks:
[FileManager +] or [Documents] 
[_] Use Default.

i dont have Default.
when i select File Manager +
select the file and Crash as soon i press Send Ok check mark.
but same file,
does Not crash, sends ok, if i send using [Documents].

problem is related to File Manager +...
File Manager + v3.0.7 apk does Not work with Warpinator v1.5.2 apk

i have Storage permisions to File Manager +, but Not contacts, No other permissions are available.

I have other File Managers apk, but Option does Not show/appear on Warpinator,

**Comment by slowscript** [comment]
I've made another attempt at fixing this. I've tried installing File Manager Plus but it doesn't offer me the option to use it to open files so I can't verify it is working. Maybe my Android 11 doesn't allow it anymore? Here is the APK: [warpinator.zip](https://github.com/slowscript/warpinator-android/files/9595006/warpinator.zip)
I would appreciate some feedback on this before I make another release that doesn't fix anything.
